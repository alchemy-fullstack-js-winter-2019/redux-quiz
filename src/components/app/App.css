* {
  font-family: 'Courier New', Courier, monospace;
  header {
    background: black;
    color: white;
    padding: 20px;
    margin: 0;
  }
  .controls {
    display: flex;
    justify-content: space-around;
    margin: 20px;
  }
  .display {
    font-family: 'Courier New', Courier, monospace;
    text-align: center;
  }
}
