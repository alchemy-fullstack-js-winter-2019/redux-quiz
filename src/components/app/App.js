import React from 'react';
import TitleSubtitleEditor from '../../containers/TitleSubtitleEditor';


function App() {
  return (
    <TitleSubtitleEditor />
  );
}

export default App;
