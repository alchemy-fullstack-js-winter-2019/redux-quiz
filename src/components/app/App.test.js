import React from 'react';
import renderer from 'react-test-renderer';
import App from './App.js';

describe('App', () => {
  it('matches a snapshot', () => {
    const tree = renderer.create(
      <App />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
