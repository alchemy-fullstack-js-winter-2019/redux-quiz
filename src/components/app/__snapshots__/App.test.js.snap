// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App matches a snapshot 1`] = `
Array [
  <form>
    <input
      name="title"
      onChange={[Function]}
      type="text"
      value=""
    />
    <input
      name="subtitle"
      onChange={[Function]}
      type="text"
      value=""
    />
  </form>,
  <h1>
     
    
     
  </h1>,
  <h2>
     
    
     
  </h2>,
]
`;
